sdk/eventhub/Azure.Messaging.EventHubs.Processor/README.md

```diff
@@ -70,13 +70,13 @@ For more concepts and deeper discussion, see: [Event Hubs Features](https://docs
 
 Since the `EventProcessorClient` has a dependency on Azure Storage blobs for persistence of its state, you'll need to provide a `BlobContainerClient` for the processor, which has been configured for the storage account and container that should be used.
 
-```csharp
-string storageConnectionString = "<< CONNECTION STRING FOR THE STORAGE ACCOUNT >>";
-string blobContainerName = "<< NAME OF THE BLOBS CONTAINER >>";
+```C# Snippet:EventHubs_Processor_ReadMe_Create
+var storageConnectionString = "<< CONNECTION STRING FOR THE STORAGE ACCOUNT >>";
```

Comment:
just curious, why the switch to `var`?

Reply:
Consistency.   I noticed that we were using `var` for the same setup across the core package.   I considered making them all `string` but after looking through some random samplings from the other service README and samples, it seemed pretty common for this kind of obvious-string with explicit name to use `var` - so I did.   

Not opposed to reconsidering and going in the other direction, if you think there's something not obvious or confusing.